src/module.c

```diff
@@ -800,7 +800,7 @@ int64_t commandFlagsFromString(char *s) {
  *                     other reason.
  * * **"no-auth"**:    This command can be run by an un-authenticated client.
  *                     Normally this is used by a command that is used
- *                     to authenticate a client. 
+ *                     to authenticate a client.
```

Comment:
please revert all whitespace-removal changes (it messes up the backlog)

Reply:
blame log..
and although these spaces should not be here, and someone else may trim them one day, since they have nothing to do with the PR and there are quite a lot of them, i also think they should be reverted. (i would discipline my editor if i where you not to do things by itself).